scripts/exregional_run_analysis.sh

```diff
@@ -408,7 +420,15 @@ cp_vrfy ${HYBENSINFO} hybens_info
 
 # Get aircraft reject list and surface uselist
 cp_vrfy ${AIRCRAFT_REJECT}/current_bad_aircraft.txt current_bad_aircraft
-cp_vrfy ${SFCOBS_USELIST}/current_mesonet_uselist.txt gsd_sfcobs_uselist.txt
+if [ -r "${SFCOBS_USELIST}/current_mesonet_uselist.txt" ]; then
+  cp_vrfy ${SFCOBS_USELIST}/current_mesonet_uselist.txt gsd_sfcobs_uselist.txt
+else
+  if [ -r "${SFCOBS_USELIST}/gsd_sfcobs_uselist.txt" ]; then
+    cp_vrfy ${SFCOBS_USELIST}/gsd_sfcobs_uselist.txt gsd_sfcobs_uselist.txt
+  else
+    print_info_msg "$VERBOSE" "Warning: gsd_sfcobs_uselist.txt  does not exist!"
+  fi
+fi
```

Comment:
Could clean up the logic and reduce duplication of info with a loop over possible files (untested example):

```
gsi_name="gsd_sfcobs_uselist.txt"
for obs_list in (${SFCOBS_USELIST}/current_mesonet_uselist.txt "${SFCOBS_USELIST}/gsd_sfcobs_uselist.txt") ; do 
  if [ -r $obs_list ] ; then
    cp_vrfy $obs_list $gsi_name
    break
  fi
fi
if [ ! -r $gsi_name ] ; then 
  print_info_msg "$VERBOSE" "Warning: $gsi_name  does not exist!"
fi
```

Reply:
I see, we can use this when we have a chance to test it.

Reply:
I updated the script based on this comment and tested it on dev2.